<?xml version="1.0"?>
<launch>

    <node pkg="wamv_driver" type="motor_command_interpretter.py" name="motor_command_interpretter_node" >
      <remap from='port_motor_input'          to='LmotorSpeed' />
      <remap from='strbrd_motor_input'        to='RmotorSpeed' />
      <remap from='port_bow_thruster_input'   to='LmotorSpeed_lateral' />
      <remap from='strbrd_bow_thruster_input' to='RmotorSpeed_lateral' />
      <remap from='motor_command'             to='motor_command' />
      <remap from='system_mode'               to='system_mode' />
    </node>

</launch>
